src/sgame/sg_bot_skilltree.cpp

```diff
@@ -160,14 +162,12 @@ static Util::optional<botSkillTreeElement_t> ChooseOneSkill(const gentity_t *bot
 std::pair<std::string, skillSet_t> BotDetermineSkills(gentity_t *bot, int skill)
 {
 	std::vector<botSkillTreeElement_t> possible_choices = initial_unlockable_skills;
-	// aliens have 51 points to spend max
+	// aliens have 61 points to spend max
 	// humans have 40 points to spend max
-	// here we give a bit more money to humans because they have more
-	// expensive skills
-	float max = G_Team(bot) == TEAM_ALIENS ? 51.0f : 46.0f;
+	float max = G_Team(bot) == TEAM_ALIENS ? 61.0f : 40.0f;
 
-	// unlock every skill at skill 8
```

Comment:
imo, we shouldn't pick a random max skill level of 7. We should pick either 5 or 10.

Reply:
Well, I mean I agree that on paper it sounds good. But now they max out at sk7 and bots are already significantly nerfed at sk5. If we did use sk10 as a max, then bots would be terribly bad by default. (and also bot skill cap at 9)

On the other side, if we did max out skills at sk5, bots would already be surprisingly professional at sk5 already and wouln't have more progression opportunities, beside being more aimbot-y. I think sk7 is fine for now. As we make bots more and more skilled the number will likely have to go further up, but for now this is reasonable enough.